src/proposal.mligo

```diff
@@ -316,18 +328,21 @@ let is_voting_period_over (proposal, voting_period, store : proposal * voting_pe
   current_period > proposal.period_num + 1n
 
 [@inline]
-let do_total_vote_meet_quorum_threshold (proposal, store, quorum_threshold : proposal * storage * quorum_threshold): bool =
+let frozen_total_supply(store : storage): nat =
+    match Map.find_opt store.frozen_token_id store.total_supply with
+    | Some v -> v
+    | None -> ((failwith "BAD_STATE") : nat)
+
+[@inline]
+let do_total_vote_meet_quorum_threshold (proposal, store, voting_period, quorum_threshold : proposal * storage * voting_period * quorum_threshold): bool =
   let votes_placed = proposal.upvotes + proposal.downvotes in
-  let total_supply =
-        match Map.find_opt store.frozen_token_id store.total_supply with
-        | Some v -> v
-        | None -> 0n
+  let total_supply = frozen_total_supply(store)
   in
   // Note: this is equivalent to checking that the number of votes placed is
   // bigger or equal than the total supply of frozen tokens multiplied by the
   // quorum_threshold proportion.
-  let reached_quorum = {numerator = votes_placed; denominator = total_supply} in
-  is_ge_qt(reached_quorum, quorum_threshold)
+  let reached_quorum = (votes_placed * abs(quorum_denominator)) / total_supply in
+  (reached_quorum > quorum_threshold)
```

Comment:
```suggestion
  (reached_quorum >= quorum_threshold)
```

Reply:
Have fixed this. Have also added a test for this, but I couldn't target an exact quorum since it is recomputed even during the first proposal.